### ISSUE TITLE
upload docker image to dockerhub

### ISSUE THREAD

**Issue by yongwen** [pinterest/snappass#13]
would be nice to create and upload the snappass docker image to dockerhub so the snappass server could be easily pulled and started, demo, etc.

**Comment by samueldg** [comment]
For now I created an automatic build on [samueldg/snappass](https://hub.docker.com/r/samueldg/snappass/).
However, the best solution to have someone with rights to the pinterest GitHub account link it with an account on the Docker hub, and do the same automated build setup to always have the latest and greatest image available.
Also, using git tags for versions would allow having different tags for the Docker repository as well.

**Comment by yongwen** [comment]
Thanks @samueldg ! and Yes, I do plan to add the auto built image to Pinterest's docker hub account so make it a little bit "official".

**Comment by nichochar** [comment]
Progress on this, @yongwen ? Is it still something we're looking to do ?

**Comment by terowz** [comment]
yes +1 on this

**Comment by samueldg** [comment]
Bumping this to let you know there's probably interest out there:
The unofficial image I have set up ([samueldg/snappass](https://hub.docker.com/r/samueldg/snappass/)) hit the 1K pulls mark recently.

**Comment by yongwen** [comment]
Super Nice!

**Comment by nichochar** [comment]
@yongwen is it possible to upload this to a `pinterest` public docker organization? '

cc @jparise

**Comment by Majed6** [comment]
Could you please upload an official image ? <3

**Comment by jparise** [comment]
We're planning on using GitHub's package registry for this once we get access to it.

**Comment by yongwen** [comment]
uploaded the latest docker image to https://github.com/pinterest/snappass/packages/20674
@jparise @Majed6 can you verify if you can pull the image: docker pull docker.pkg.github.com/pinterest/snappass/snappass:4acef097e8ad64933e53c17ca5f1144bacc4859f

Not very sure how the default docker pull permission is for public consumption, as it seems to require docker login. this is the help page for using github registry https://help.github.com/en/articles/configuring-docker-for-use-with-github-package-registry

**Comment by jparise** [comment]
Yes, it looks like GitHub's Docker registry requires authentication. Given that, we might be better off with a public image on Docker Hub.

**Comment by yongwen** [comment]
yeah looks like it. Maybe something we could give feedback to Github package team to support public docker registry for public repo.

**Comment by jgournet** [comment]
> Yes, it looks like GitHub's Docker registry requires authentication. Given that, we might be better off with a public image on Docker Hub.

@jparise and @yongwen : github's docker registry does not require authentication anymore (for quite some time by now).
=> any chance you could push your image to it now ?

Note: thank you for the work on this application !
